### PR TITLE
WIP: better responsiveness

### PR DESCRIPTION
**Pull request by Keitio**
WIP means it could be better, but it should be ready to merge

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,10 @@
     "node": "16.x"
   },
   "dependencies": {
+    "@reach/dialog": "^0.16.0",
+    "@reach/listbox": "^0.16.1",
     "@reach/tabs": "^0.16.1",
+    "@reach/visually-hidden": "^0.16.0",
```
Comment (KeziahMoselle) [inline package.json:20]: Are we still using this package ?

File: src/components/decorations/HR.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import SVG from "react-inlinesvg";
+
+export default function HR(): JSX.Element {
+  return <img src="/border.svg" alt="" />;
```
Comment (KeziahMoselle) [inline src/components/decorations/HR.tsx:4]: You are not using the `<SVG>` component imported above ?

File: src/pages/characters.tsx [hunk 3]
```diff
@@ -1,168 +1,13 @@
...
-            </div>
-          ))}
-        </div>
-      ))}
-    </div>
-  );
-}
+import CharacterCostumes from "@components/characters/CharacterCostumes";
+import CharacterRows from "@components/characters/CharacterRows";
+import CostumeSelect from "@components/characters/CostumeSelect";
```
Comment (KeziahMoselle) [inline src/pages/characters.tsx:10]: 👍 really nice refactor

File: src/styles/index.scss [hunk 4]
```diff
@@ -6,7 +6,17 @@ html,
...
+}
+
+[data-reach-dialog-overlay] {
+  color: black;
+  background: white;
+  width: 100%;
+}
+[data-reach-dialog-content] {
+  width: 90% !important;
+  // height: 90%;
```
Comment (KeziahMoselle) [inline src/styles/index.scss:19]: is it something useful for later ?

**Review by KeziahMoselle — APPROVED** [review]

File: src/components/decorations/HR.tsx (hunk 2 above)
Comment (Keitio) [reply src/components/decorations/HR.tsx:4]: Nope, my bad

File: src/styles/index.scss (hunk 4 above)
Comment (Keitio) [reply src/styles/index.scss:19]: Nope, forgot to delete it, was for a test

File: package.json (hunk 1 above)
Comment (Keitio) [reply package.json:20]: nope

**Pull request merged**